master/src/main/java/eu/unifiedviews/master/api/DPUResource.java

```diff
@@ -96,10 +106,6 @@
 
     private void replaceDpu(DPUTemplateRecord dpuTemplate, File jarFile) {
         try {
-            List<Pipeline> pipelines = pipelineFacade.getPipelinesUsingDPU(dpuTemplate);
-            if(!pipelines.isEmpty()) {
-                throw new ApiException(Response.Status.CONFLICT, "DPU is already used in pipelines!.");
```

Comment:
User should be able to replace DPU even if it is already used in pipelines. ReplaceDPU method is only called when force flag is set, and thus user should be aware what it could be potentially dangerous.